language/English/strings.po

```diff
@@ -15548,7 +15687,10 @@ msgctxt "#36419"
 msgid "Define locations used for retrieving weather information."
 msgstr ""
 
-#empty string with id 36420
+#: system/settings/settings.xml
```

Comment:
please add a context sensitive note to which setting this description belongs - thanks.